static/scripts/upload_files.js

```diff
@@ -0,0 +1,29 @@
+﻿async function uploadTestFile() {
+    try {
+        // Step 1: Fetch the signed URL from the Go API
+        const response = await fetch("https://your-api.com/get-signed-url");
+        if (!response.ok) throw new Error("Failed to get signed URL");
+
+        const { url } = await response.json(); // Extract signed URL
+
+        // Step 2: Create a test file as a Blob (Example: a simple text file)
+        const testFile = new Blob(["Test file content"], { type: "text/plain" });
```

Comment:
Should be using this function:

https://github.com/LivingInSyn/dlptest/blob/main/static/scripts/common.js#L1